makefiles/tools/nrfjprog.inc.mk

```diff
@@ -0,0 +1,11 @@
+NRFJPROG ?= nrfjprog
+FLASHER ?= $(NRFJPROG)
+#nrfjprog only support hexfile
+FLASHFILE = $(HEXFILE)
+FFLAGS ?= --program $(FLASHFILE)
+
+# Give the possibility to user to flash nRF modem firmware
+# To do so, use make nrf_mode/flash_firmware
+ifneq (,$(filter nrf_modem, $(USEPKG)))
```

Comment:
Also, we could make this independent of the package being required right? It's just flashing the modem firmware